test/functional/fixtures/api/raw/wait-for-element/test.js

```diff
@@ -0,0 +1,27 @@
+var expect = require('chai').expect;
+
+describe('[Raw API] Wait for element action', function () {
+    it("Should pass if the wait-for-element command's timeout exceeds time required for the element to appear", function () {
```

Comment:
We also need to test scenario then element appears after page load: e.g. create page that redirects after some timeout to the page with element. Run `waitForElement` with timeout bigger then the one that is for the redirect and test action completion.